docs/user-documentation/searching.md

```diff
@@ -0,0 +1,60 @@
+
+Once the fields are added they can be configured further on the 'Fields' tab, although the label, machine name, and type usually don't need to be changed. The 'Type' setting has several different types of Full-text processing available, which may be of interest. Each is described in the expandable 'Data Types' box at the bottom of the page. The 'Boost' setting allows repository managers to increase the weight of particular fields when calculating search relevancy.
+
+#### Processors Tab
+
+The 'Processors' tab allows repository managers to adjust how data and search queries are processed to adjust results. The defaults are acceptable in most cases.
+
+## Searching Islandora 8
+
+The Search API SOLR module comes with a search page pre-configured (accessible at '/solr-search/content'). To edit the search page, navigate to '/admin/structure/views/view/solr_search_content'.
+
+Repository managers may want to change the URL used to access the page, add it to the site navigation, or add a search box. In the 'Page Settings' box in the middle of the page, click on the existing path to open a shadow-box with an edit field. Change the URL as desired, for example, to 'search' and click 'Apply'. Then, click the 'No menu' link just below it to open the menu settings shadow-box. Selecting 'Normal menu entry' will allow a repository manager to add a menu link text, description, and place it within the site menu tree (the default, `<Main navigation>` works for most sites). A search box can be added by expanding the 'Advanced' options and changing the 'Exposed form in block' setting and then use the 'Block Layout' interface to place the block where desired. After making changes to the View's settings, click the 'Save' button to ensure the changes are not lost.
+
+Islandora's Repository Items are displayed in the search results as a fully rendered entity by default. Repository managers can choose which view mode should be used for each search datasource by clicking the 'Settings' link next to the 'Show:' setting under the 'Format' section of the search view configuration page. The 'Teaser' and 'Search result highlighting input' are the two most likely options. Alternatively, repository managers can select specific fields to display instead by clicking the 'Rendered Entity' link and changing it to 'Fields' and then choosing which fields will be displayed in the 'Fields' section underneath. **Note:** thumbnails will not immediately be available using this method without more advanced configurations.
```

Comment:
Recommend a screenshot to show where 'Format' section and  'Settings link' is .

Reply:
@seth-shaw-unlv all done :)

Reply:
> **Note:** thumbnails will not immediately be available using this method without more advanced configurations.

This also sounds like an issue in disguise as documentation :)

Reply:
@dannylamb this is what we get for using EVA to display thumbnails and service files; they don't show up as available fields in the Views UI. It *should* be possible with Relationships under the Advanced section, but I haven't gotten it to work (it was easier to define a display mode).

Reply:
I gotcha.  That limitation is just for displaying fields and not a rendered entity.  Good to know.